src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -165,7 +165,7 @@
 
   position: relative;
   display: inline-flex;
-  align-items: center;
+  align-items: baseline;
```

Comment:
It looks like we're using `align-items: center` on the dropdown toggle - I'm curious if you know what breaks when we use that here in the dropdown? It seems like we would (almost?) always want center aligned items. I know we've gone back and forth on this, but I'm curious where we landed and if we might be able to center align these items again, or if maybe the dropdown needs to be updated to match since they're so similar.